openpype/plugins/publish/integrate_new.py

```diff
@@ -139,6 +139,10 @@ def process(self, instance):
                     ef, instance.data["family"], instance.data["families"]))
                 return
 
+        # instance should be published on a farm
+        if instance.data["farm"]:
```

Comment:
shouldn't this be checked more carefully? What if instance doesn't have `farm` (as it happened to me right now)